lib/checker/checkercomponent.cpp

```diff
@@ -162,13 +164,26 @@ void CheckerComponent::CheckThreadProc()
 
 			TimePeriod::Ptr tp = checkable->GetCheckPeriod();
 
-			if (tp && !tp->IsInside(Utility::GetTime())) {
-				Log(LogNotice, "CheckerComponent")
-					<< "Skipping check for object '" << checkable->GetName()
-					<< "': not in check period '" << tp->GetName() << "'";
+			if (tp) {
+				auto ts (Utility::GetTime());
+				ObjectLock oLock (tp);
 
-				check = false;
-				notifyNextCheck = true;
+				if (!tp->IsInside(ts)) {
+					nextCheck = tp->FindNextTransition(ts);
+
+					if (nextCheck <= 0) {
+						reschedulingCheckLevel = LogInformation;
```

Comment:
I'm not sure why that log level promotion is needed here, since this case is nothing special. Yes, @alvar did suggest changing the log level when `FindNextTransition()` yield nothing with https://github.com/Icinga/icinga2/pull/10070#discussion_r1926695558, but with the current state of the PR the checkable is going to be rescheduled within the next `24h` anyway, so I wouldn't clutter the info level with such useless log.